go.mod

```diff
@@ -1,87 +1,191 @@
+	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a // indirect
+	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/oauth2 v0.0.0-20210622215436-a8dc77f794b6 // indirect
+	golang.org/x/sys v0.0.0-20210817190340-bfb29a6856f2 // indirect
+	golang.org/x/text v0.3.6 // indirect
+	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
+	golang.org/x/tools v0.1.2 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
+	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c // indirect
+	google.golang.org/grpc v1.38.0 // indirect
+	google.golang.org/protobuf v1.26.0 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	k8s.io/component-base v0.22.2 // indirect
+	k8s.io/klog v1.0.0 // indirect
+	k8s.io/klog/v2 v2.9.0 // indirect
+	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
+)
+
 // Pinned to kubernetes-1.21.3
 replace (
 	// see https://github.com/rook/rook/blob/master/go.mod#L42-L43
 	github.com/moby/term => github.com/moby/term v0.0.0-20201110203204-bea5bbe245bf // indirect
 	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3
-	k8s.io/api => k8s.io/api v0.21.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.3
-	k8s.io/apiserver => k8s.io/apiserver v0.21.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.21.3
-	k8s.io/client-go => k8s.io/client-go v0.21.3
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.21.3
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.21.3
-	k8s.io/code-generator => k8s.io/code-generator v0.21.3
-	k8s.io/component-base => k8s.io/component-base v0.21.3
-	k8s.io/component-helpers => k8s.io/component-helpers v0.21.3
-	k8s.io/controller-manager => k8s.io/controller-manager v0.21.3
-	k8s.io/cri-api => k8s.io/cri-api v0.21.3
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.21.3
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.21.3
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.21.3
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.21.3
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.21.3
-	k8s.io/kubectl => k8s.io/kubectl v0.21.3
-	k8s.io/kubelet => k8s.io/kubelet v0.21.3
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.3
-	k8s.io/metrics => k8s.io/metrics v0.21.3
-	k8s.io/mount-utils => k8s.io/mount-utils v0.21.3
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.3
-
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
```

Comment:
This is a hack to fix openapi generated code. Ideally, we should fix generation code.
Proper fix would probably require using new sdk and other generation tools.